superset-frontend/cypress-base/cypress.json

```diff
@@ -10,5 +10,5 @@
   "videoUploadOnPasses": false,
   "viewportWidth": 1280,
   "viewportHeight": 1024,
-  "projectId": "dk2opw"
+  "projectId": "ukwxzo"
```

Comment:
Created a new project to clear up histories from the test runs.

Reply:
We should prob leave this for now..

Reply:
The current project id in code has already been deleted... Since this is not used by anyone other than me, it's probably OK. I can also just delete this altogether.